drivers/i2s/i2s_mcux.c

```diff
@@ -0,0 +1,1209 @@
+/*
```

Comment:
I think this file should probably be renamed to `i2s_mcux_sai.c` to be clear this is using the SAI block.

Reply:
ok